docs/content/latest/api/ycql/ddl_create_table.md

```diff
@@ -160,6 +160,14 @@ cqlsh:example> SELECT * FROM devices;
            1 |       2 | Acme Corporation | Giant Rubber Band
 ```
 
+### Use table property to define the number of tablets
+
+You can do this as shown below.
+
+```cassandraql
+cqlsh:example> CREATE TABLE users(user_id INT PRIMARY KEY, full_name TEXT) WITH tablets = 10;
+```
```

Comment:
I noticed that the grammar is incorrect for table_options, please fix that too. It should be
```
CREATE TABLE ... [WITH table_options]
table_options = table_property [AND table_property ...]
```